pkg/capacity/topology/doc.go

```diff
@@ -0,0 +1,133 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package topology contains an abstract interface for discovering
+// topology segments for a storage backend and a specific implementation
+// which does that based on the CSINodeDriver.TopologyKeys and the
+// corresponding labels for the nodes.
+package topology
+
+import (
```

Comment:
Anything below this line does not belong to `topology/doc.go`

Reply:
Okay. I wasn't sure whether doc.go was just for package-level documentation or also for interfaces of the package. I'll move it.